### PR TITLE
feat(BuildTranslation): add build-translations package

### PR DESCRIPTION
**Pull request by allison-c**
### Purpose 🚀
_clear, one sentence description of primary purpose of this PR_

### Notes ✏️
See https://github.com/acl-services/paprika/blob/13830fe609634d7f3ed475006d605cb157276781/packages/BuildTranslations/README.md

### Updates 📦
If you have changed a component's source code (not stories, specs, or docs), before merging your branch run `yarn changeset`. This will prompt you to:
- indicate if changes were patch/minor/major for each modified package
- enter a release message

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/your-branch-name

### Screenshots 📸
_optional but highly recommended_

### References 🔗
_relevant Jira ticket / GitHub issues_

### CODE REVIEW

File: packages/BuildTranslations/src/cli.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  return {
+    sourcePath,
+    outputPath: args["--out-dir"],
+    yamlFileExtension: args["--yaml-file-ext"],
+  };
+}
+
+export async function cli(args) {
+  const options = parseArgumentsIntoOptions(args);
+  await buildTranslations(options);
```
Comment (nahumzs) [inline packages/BuildTranslations/src/cli.js:26]: does it worth adding a message at the end? `console.log("locales have been processed")`

**Comment by nahumzs** [comment]
@mikrotron ESL question, what is a better name for the package build-locale or build-translations?

**Comment by nahumzs** [comment]
what about `L10nTransformer` as name @allison-c ? @mikrotron suggestion :D

**Comment by allison-c** [comment]
I'm adding some tests, but it turns out not possible to mock `process.cwd()` in jest, see https://github.com/facebook/jest/issues/5146
I'm using it a lot in the script, can I skip writing tests this time?

**Comment by nahumzs** [comment]
just add a small mention on the readme page about it (testing) 🙏 this is a package that will fail really early so if something is not working we will figure out quick easy without disturbing users or devs work too much.

**Review by nahumzs — APPROVED** [review]

**Pull request merged**